variantopedia/templates/variantopedia/health_check_details.html

```diff
@@ -0,0 +1,64 @@
+{% load js_tags %}
+{% load static %}
+{% load classification_tags %}
+{% load ui_utils %}
+<div class="container">
+    <div class="card">
+        <div class="card-header">Recent Activity</div>
+        <div class="card-body">
+            {% for line in recent_lines %}
+            <span>{{ line|safe }}</span>
+                {% if not forloop.last %}<hr>{% endif %}
+            {% endfor %}
+        </div>
+    </div>
+
+    {% for name, previews_list in previews.items %}
+        <div class="card mt-4">
+            <div class="card-header">{{ name }}</div>
+            <div class="card-body">
+                {% for preview in previews_list %}
+                    <div class="row">
+                        <div class="col-3">
+                            <p><i class="{{ preview.icon }}"></i> {{ preview.category }}</p>
+                        </div>
+                        <div class="col-4">
+                            <p><a href="{{ preview.internal_url }}">{% if name == 'Active Users' %}{{ preview.identifier }} {% else %}{{ preview.title }}{% endif %}</a></p>
+                        </div>
+                        {% if name == 'Active Users' %}
+                            <div class="col-5">
+                                <a class="toggle-link" data-toggle="collapse" href="#active-users{{ preview.obj }}">View Events</a>
```

Comment:
When preview.obj is converted to a string, it can contain characters not safe for HTML ids.
There is a method in ui_utils `id_safe` which calls `html_id_safe` so you can call {{ preview.obj | id_safe }}. The method now only works on strings, but you could update the method `html_id_safe` to just wrap `text = str(text)` after the none check.